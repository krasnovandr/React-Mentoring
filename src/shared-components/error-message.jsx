import React from "react";
import Grid from "@material-ui/core/Grid";

export function ErrorMessage() {
    return (
        <Grid container spacing={24} direction="column" alignItems="center" justify="center">
            <Grid item xs={12} >
                <h1>Something went wrong. Please contact the asministrator.</h1>
            </Grid>
        </Grid>
    )
}
