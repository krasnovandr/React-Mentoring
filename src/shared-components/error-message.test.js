import React from 'react'
import { render, cleanup } from 'react-testing-library'
import { Route, Link, MemoryRouter } from "react-router-dom";
import Header from "./header"
import renderer from 'react-test-renderer';
import { ErrorMessage } from './error-message';
// afterEach(cleanup)
describe('Error  Component', () => {
    it('should render error component', () => {
        const component = renderer.create(
            <ErrorMessage />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    })
})