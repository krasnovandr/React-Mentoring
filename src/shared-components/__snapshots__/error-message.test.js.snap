// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error  Component should render error component 1`] = `
<div
  className="MuiGrid-container-1 MuiGrid-spacing-xs-24-25 MuiGrid-direction-xs-column-4 MuiGrid-align-items-xs-center-9 MuiGrid-justify-xs-center-18"
>
  <div
    className="MuiGrid-item-2 MuiGrid-grid-xs-12-41"
  >
    <h1>
      Something went wrong. Please contact the asministrator.
    </h1>
  </div>
</div>
`;
